demo/docker-compose.yml

```diff
@@ -156,6 +165,7 @@ services:
       - WAIT_SLEEP_INTERVAL=1
       - WAIT_HOST_CONNECT_TIMEOUT=60
       - TUNNEL_ENDPOINT=http://mediator-tunnel:4040
```

Comment:
Should `TUNNEL_ENDPOINT` be removed?

Reply:
I wasn't sure so I left it in.  You can run the demo with or without the tunnel (today the tunnel was working for me, even though last week it wasn't)